src/execution.jl

```diff
@@ -656,9 +663,10 @@ macro btime(args...)
     return esc(
         quote
             local $bench = $BenchmarkTools.@benchmarkable $(args...)
-            $BenchmarkTools.warmup($bench)
             $tune_phase
-            local $trial, $result = $BenchmarkTools.run_result($bench)
+            local $trial, $result = $BenchmarkTools.run_result(
+                $bench; warmup=$(hasevals(params))
```

Comment:
same remark on tuning

Reply:
I also think my same response applies here. Also note that both cases are also running in a macro.